src/controller/CHIPDeviceController.cpp

```diff
@@ -848,16 +874,13 @@ void DeviceCommissioner::OnSessionEstablished()
 
     ChipLogDetail(Controller, "Remote device completed SPAKE2+ handshake");
 
+    ChipLogProgress(Controller, "OnPairingComplete");
```

Comment:
This log is stale. Could we combined with previous?

Reply:
removed.